### PR TITLE
:bug: Write errors to stderr

### LINKED ISSUE #479: [BUG] send errors to stderr
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Konveyor version

latest

### Priority

Major

### Current Behavior

errors are logged like so https://github.com/konveyor/analyzer-lsp/blob/main/cmd/analyzer/main.go#L100 

### Expected Behavior

_No response_

### How Reproducible

Always (Default)

### Steps To Reproduce

_No response_

### Environment

_No response_

### Anything else?

In kantra, we have a log file for analyzer logs. There is an issue then with a user hitting an error and it not being apparent. We should send these errors to stderr as well.

### PR DESCRIPTION
**Pull request by eemcmullan**
Closes #479

### CODE REVIEW

**Comment by shawn-hurley** [comment]
is there a way to not remove the errors from the other log/logfile as well?

**Review by shawn-hurley — APPROVED** [review]

**Comment by pranavgaikwad** [comment]
@eemcmullan needs rebase

**Review by aufi — APPROVED** [review]
LGTM

File: cmd/analyzer/main.go [hunk 1]
```diff
@@ -219,6 +230,7 @@ func main() {
 	err = os.WriteFile(outputViolations, b, 0644)
 	if err != nil {
 		log.Error(err, "error writing output file", "file", outputViolations)
+		errLog.Error(err, "error writing output file", "file", outputViolations)
```
Comment (pranavgaikwad) [inline cmd/analyzer/main.go:233]: @eemcmullan is this printing error twice on the console? if yes, why not only keep stderr

**Review by pranavgaikwad — CHANGES_REQUESTED** [review]

File: cmd/analyzer/main.go (hunk 1 above)
Comment (shawn-hurley) [reply cmd/analyzer/main.go:233]: @pranavgaikwad I asked that we make sure that the error still went to the log file and not just to Stderr. if this is printing twice @eemcmullan I think you probably need to look for a MultiWriter for the file and stdout when a file is set up: https://github.com/Sirupsen/logrus/issues/230#issuecomment-381639138.

**Review by pranavgaikwad — APPROVED** [review]

**Pull request merged**